### ISSUE TITLE
Figure out how to limit our public display's interactive capacity

### ISSUE THREAD

**Issue by devinhalladay** [GD399-OSD/open-for-discourse#4]
More specifically, we want to be able to make the OS running on the Raspberry Pi kind of just disappear so that it's just a web browser running our forthcoming public display website. This way we can leave a mouse and keyboard connected to the pi and not worry about people doing crazy stuff or screwing with our setup, but they'll still be able to interact with the website.

**Comment by bjornmeansbear** [comment]
Yeah, a headless option that boots directly to a fullscreen browser w/ no other OS would work... I have no clue how to do this, but will be happy to look for ideas

**Comment by KHurleyDesign** [comment]
we have a couple websites in __Counternarratives__ and we used a setting to prevent people from going to any other site/ leaving the website. I am not 100% sure how they did this because it was not my role in the install but I should be able to find out. Doesn't solve the overall problem but might help as a temporary fix.
